openspec/changes/archive/2026-02-07-add-mcp-package/design.md

```
@@ -0,0 +1,70 @@
+## Context
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)